### ISSUE TITLE
relationships aren't marked as required

### ISSUE THREAD

**Issue by glentakahashi** [graphql-python/graphene-sqlalchemy#251]
given this code:

```
class Parent(db.Model):
    __tablename__ = "parent"
    id = db.Column(Integer, primary_key=True)

    children = db.relationship("Child", back_populates="parent")

class Child(db.Model):
    __tablename__ = "child"
    id = db.Column(Integer, primary_key=True)

    parent_id = db.Column(Integer, ForeignKey("parent.id"), nullable=False)
    parent = db.relationship("Parent", back_populates="children")
```

I would expect this to be generated:

```
type Parent {
  id: ID!
  children: [Child!]!
}
type Child {
  id: ID!
  parent: Parent!
}
```

However this is what gets generated:

```
type Parent {
  id: ID!
  children: [Child]
}
type Child {
  id: ID!
  parent: Parent
}
```

**Comment by adrianschneider94** [comment]
This is a important feature because it simplifies writing the frontend code.
I use the graphql code generator with typescript and all I have to consider every time when I use a relationship, if a value could be null or not.

**Comment by this-vidor** [comment]
This seems like fundamental missing functionality. How has this received no attention from devs since 2019?

**Comment by erikwrede** [comment]
resolved in #367 for `:n`relationships. As argued in the PR review, we cannot reliably determine the nullability or non-nullability for `:1`-relationships just based on the SQLAlchemy model. Client controlled nullability will resolve this in the future. For now, you can customize the field type of `:1`relationship fields using the `ORMField`.
